client-admin/src/components/conversation-admin/conversation-has-comments-check.js

```diff
@@ -48,4 +48,10 @@ class ConversationHasCommentsCheck extends React.Component {
   }
 }
 
+ConversationHasCommentsCheck.defaultProps = {
+  accepted_comments: [],
+  rejected_comments: [],
+  unmoderated_comments: [],
+};
```

Comment:
Note that one more thing I had to do to stabilize e2e tests was ensure default props existed for these vars. Seems that under some conditions, component tried to render when one of these was `null`, and cypress was erroring out due to console error/warning.

This should be an improvement anyhow :)

Reply:
See this output for the error that was consistently getting thrown when loading the share page:
https://github.com/pol-is/polisServer/runs/813229303?check_suite_focus=true#step:6:127